src/lsdb/io/parquet_io.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from hipscat.io import FilePointer
+from pyarrow import Schema
+import pyarrow.parquet as pq
```

Comment:
There are similar methods for both of these in `hipscat.io.file_io`. Are you planning to add more here that would necessitate having a separate io layer?

If we should move to the hipscat io methods, that can wait for a follow-up refactor.

Reply:
Yeah, this was written before hipscat had those methods, and I originally thought that hipscat wouldn't need to touch parquet files. But these should be replaced now they're in hipscat.

Reply:
Created an issue for this #20